src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -93,6 +93,10 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
 
   const hasRoomEvents = renderedRoomEvents?.length > 0;
 
+  const iconStyles = {
+    backgroundImage: `url(${room.image_url})`,
```

Comment:
Something like this/similar is probably better so we don't render the style as `url(undefined)`:

```suggestion
    backgroundImage: room.image_url ? `url(${room.image_url})` : undefined,
```